protos/Markers/MarkerBase.proto

```diff
@@ -33,7 +118,7 @@ PROTO MarkerBase [
       }
     ]
     name IS name
-    model IS model
+    model %<= "\"" + uid + "_" + fields.model.value + "_base\"" >%
```

Comment:
This looks a lot like it will create unique identifiers for each marker, potentially providing competitors with a means (at least within a given match and for tokens they've already interacted with) to infer the kind of token from its marker. That seems undesirable given that not being able to do that was one of the core design principles of the game.

Reply:
This is a unique value for each token so that the additional points can be linked to the correct token to emulate occlusion. At this level of the API the students could equally just call getNode to get the unique node identifier of every instance of a marker.